### ISSUE TITLE
Hotspot duplicate location error

### ISSUE THREAD

**Issue by yourKatharsis** [chamilo/chamilo-lms#2813]
**Describe**
When solving a hotspot-test and klick different numbers on the same area, the previous numbers move to the left top corner.

**To Reproduce**
1. Create a hotspot-test with at least 2 areas
2. try to solve and click two times on the same location
3. submit
4. on the results page you see that the first number is now in the left top corner (see screenshot)

**Expected behavior**
It should be possible to set more than one number to a location.

**Screenshots**
![image](https://user-images.githubusercontent.com/29522268/53155475-70426000-35bd-11e9-90f4-3e8db953e7e4.png)

**Desktop (please complete the following information):**
 - OS: Windows 8.1
 - Browser chrome  72.0.3626.109

**Server
 - OS: Linux Ubuntu 16.04
 - Version of Chamilo:  1.11.8
 - Version of PHP:  7.0

**Additional context**

Apache Error Log:
```
[Thu Feb 21 10:35:07.367896 2019] [:error] [pid 7496] [client 000.00.00.00:0000] PHP Notice:  Undefined variable: correctAnswerId in /var/www/chamilo/www/main/exercise/exercise.class.php on line 4749, referer: http://xxx/main/exercise/exercise_submit.php?cidReq=TESTKURS&id_session=0&gidReq=0&gradebook=0&origin=learnpath&exerciseId=1&learnpath_id=3&learnpath_item_id=17&learnpath_item_view_id=84
[Thu Feb 21 10:35:10.709654 2019] [:error] [pid 7492] [client 000.00.00.00:0000] PHP Notice:  Undefined variable: correctAnswerId in /var/www/chamilo/www/main/exercise/exercise.class.php on line 4749, referer: http://xxx/main/lp/lp_controller.php?cidReq=TESTKURS&id_session=0&gidReq=0&gradebook=0&origin=learnpath&action=view&lp_id=3&lp_item_id=17&exeId=261&fb_type=0
```

**Comment by ywarnier** [comment]
Thanks for reporting @yourKatharsis 
I'm moving this issue to 2.0 because of its relatively limited impact. Hopefully this doesn't impact you too much, but if it does remember you can support the project and get it done faster by asking an official provider https://chamilo.org/providers

**Comment by ywarnier** [comment]
Issue #4315 should be fixed before this

**Comment by ywarnier** [comment]
#4315 is fixed now

**Comment by ywarnier** [comment]
https://github.com/chamilo/chamilo-lms/pull/4493 should have fixed the issue for upcoming version 2.0

**Comment by ywarnier** [comment]
This is fixed in 2.0. There are still issues with the visualization of hostpot prompts, but this is for another issue.
